asapdiscovery-data/asapdiscovery/data/openeye.py

```diff
@@ -148,6 +148,44 @@ def load_openeye_pdb(
         oechem.OEThrow.Fatal(f"Unable to open {pdb_fn}")
 
 
+def load_openeye_smi(smi_fn: Union[str, Path]) -> oechem.OEGraphMol:
```

Comment:
Single smiles only? probably better to have it return all the ones in a file, I know there is a bit of a historical artifact tho this, but still would probably prefer loading all the mols.